CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+### 1.11.1
```

Comment:
I added a comment in the other PR too about having the same line with the link for the release page like in `1.11.01